web/src/app/page.tsx

```diff
@@ -1,50 +1,58 @@
 import { HeroSection } from "@/components/hero"
 import { RecentlyAddedIcons } from "@/components/recently-added-icons"
-import { BASE_URL, REPO_NAME, getDescription, websiteTitle } from "@/constants"
+import {
+	BASE_URL,
+	DEFAULT_KEYWORDS,
+	DEFAULT_OG_IMAGE,
+	GITHUB_URL,
+	ORGANIZATION_NAME,
+	ORGANIZATION_SCHEMA,
+	REPO_NAME,
+	SITE_NAME,
+	SITE_TAGLINE,
+	WEB_URL,
+	getHomeDescription,
+	websiteFullTitle,
+	websiteTitle,
+} from "@/constants"
 import { getRecentlyAddedIcons, getTotalIcons } from "@/lib/api"
 import type { Metadata } from "next"
 
 export async function generateMetadata(): Promise<Metadata> {
 	const { totalIcons } = await getTotalIcons()
+	const description = getHomeDescription(totalIcons)
 
 	return {
 		title: websiteTitle,
-		description: getDescription(totalIcons),
-		keywords: ["dashboard icons", "service icons", "application icons", "tool icons", "web dashboard", "app directory"],
+		description,
+		keywords: DEFAULT_KEYWORDS,
 		robots: {
 			index: true,
 			follow: true,
 		},
 		openGraph: {
-			title: websiteTitle,
-			description: getDescription(totalIcons),
+			title: websiteFullTitle,
+			description,
 			type: "website",
-			url: BASE_URL,
-			images: [
-				{
-					url: "/og-image.png",
-					width: 1200,
-					height: 630,
-					alt: "Dashboard Icons",
-				},
-			],
+			url: WEB_URL,
+			images: [DEFAULT_OG_IMAGE],
 		},
 		twitter: {
-			title: websiteTitle,
-			description: getDescription(totalIcons),
+			title: websiteFullTitle,
+			description,
 			card: "summary_large_image",
-			images: ["/og-image.png"],
+			images: [DEFAULT_OG_IMAGE.url],
 		},
 		alternates: {
-			canonical: BASE_URL,
+			canonical: WEB_URL,
 		},
 	}
 }
 
 async function getGitHubStars() {
 	const response = await fetch(`https://api.github.com/repos/${REPO_NAME}`)
 	const data = await response.json()
-	console.log(`GitHub stars: ${data.stargazers_count}`)
+	// TODO: Consider caching this result or fetching at build time to avoid rate limits.
```

Comment:
Why remove console log + it's already cached by default with nextjs

Reply:
Biome removed it. Is it really cached? No caching on local builds.

Reply:
After it's built it won't be request again, the data is hard-coded into the page at build time